@ecomplus/storefront-components/src/scss/TheCart.scss

```
@@ -0,0 +1,55 @@
+.cart {
+    &__empty {
```

```suggestion
  &__empty {
```

A identação foi quebrada daqui pra baixo também...